File: src/tests/fuzz_corpus/seed.go
package main

import (
	"fmt"
	"net/http"
	"os"
)

type Config struct {
	Host string
	Port int
}

type Server struct {
	config Config
}

func NewServer(cfg Config) *Server {
	return &Server{config: cfg}
}

func (s *Server) Start() error {
	addr := fmt.Sprintf("%s:%d", s.config.Host, s.config.Port)
	return http.ListenAndServe(addr, nil)
}

func loadEnv(key string) string {
	return os.Getenv(key)
}

File: src/tests/fuzz_corpus/seed.dart
import 'dart:io';
import 'package:flutter/material.dart';

class Config {
  final String host;
  final int port;
  Config({required this.host, required this.port});
}

enum AppState { loading, ready, error }

mixin Logging {
  void log(String msg) => print(msg);
}

class Server with Logging {
  final Config config;
  Server(this.config);

  Future<void> start() async {
    log('Starting on ${config.host}:${config.port}');
  }
}

void main() {
  final server = Server(Config(host: 'localhost', port: 8080));
  server.start();
}
